docs/AppObservability.md

```diff
@@ -158,7 +158,7 @@ curl -s https://raw.githubusercontent.com/retaildevcrews/helium/main/docs/dashbo
 sed -i "s/%%SUBSCRIPTION_GUID%%/$(eval $He_Sub)/g" Helium_Dashboard.json
```

Comment:
Are these valid for an AKS deployment?  The previous dashboards were based on App Services reporting to App Insights.